bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -101,28 +105,34 @@ public BigtableDataGrpcClient(
     this.channel = channel;
     this.executorService = executorService;
     this.retryOptions = retryOptions;
+    setClientCallService(ClientCallService.DEFAULT);
+  }
 
-    this.sampleRowKeysAsync = BigtableAsyncUtilities.createSampleRowKeyAsyncReader(this.channel);
-    this.readRowsAsync = BigtableAsyncUtilities.createRowKeyAysncReader(this.channel);
+  @VisibleForTesting
+  void setClientCallService(ClientCallService clientCallService) {
+    this.clientCallService = clientCallService;
+    this.sampleRowKeysAsync =
+        BigtableAsyncUtilities.createSampleRowKeyAsyncReader(this.channel, clientCallService);
+    this.readRowsAsync =
+        BigtableAsyncUtilities.createRowKeyAysncReader(this.channel, clientCallService);
   }
 
   @Override
   public Empty mutateRow(MutateRowRequest request) throws ServiceException {
-    return ClientCalls.blockingUnaryCall(
+    return clientCallService.blockingUnaryCall(
       channel.newCall(BigtableServiceGrpc.METHOD_MUTATE_ROW, CallOptions.DEFAULT), request);
```

Comment:
The indenting is now in line with all of the other similar calls.  Eclipse does it this way.  Can we keep it as is?